.github/workflows/05-prepare-release.yml

```
@@ -79,7 +79,7 @@ jobs:
   split:
     needs: build
     runs-on: ubuntu-24.04
-    environment: ${{ format('manyrepos ({0})', github.ref_type) }}
+    environment: ${{ format('manyrepos_', github.ref_type) }}
```

```suggestion
    environment: ${{ format('manyrepos_{0}', github.ref_type) }}
```

Does format not need a "placeholder"?